lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -263,8 +290,13 @@ def get_d3_sequential_open_distrib(course_id):
             subsection_name = own_metadata(subsection).get('display_name', '')
 
             num_students = 0
-            if subsection.location in sequential_open_distrib:
-                num_students = sequential_open_distrib[subsection.location]
+
+            try:
+                module = client.modules(course_id, subsection.location)
```

Comment:
The API and client do not have methods/endpoints which accept a list of module_id's at the moment.  I think that's a good idea to add, but I don't think I should put this PR on hold for it.  I can definitely implement bulk module fetching if it's important for people using the client, but I think that's an issue for another PR.